staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2024,9 +2024,10 @@ type EnvVar struct {
 	// Escaped references will never be expanded, regardless of whether the variable
 	// exists or not.
 	// Defaults to "".
```

Comment:
The comment on line 2016 needs to be removed since we are making these comments inside the field description.